dotcom-rendering/src/palette.ts

```diff
@@ -49,6 +49,24 @@ const pillarPalette = (
 	}
 };
 
+const textblockTextLight: PaletteFunction = (format: ArticleFormat) => {
+	switch (format.design) {
+		case ArticleDesign.Audio:
+			return sourcePalette.neutral[97];
+		default:
+			return sourcePalette.neutral[7];
```

Comment:
As discussed, a safer default here might be
```suggestion
			return 'inherit';
```